### DISCUSSION TITLE
missing parameter offset in  projector plugin

### DISCUSSION THREAD

**Discussion by Jackson334-prog** [mitsuba-renderer/mitsuba3#354] [Q&A]
I notice that there is some parameters miss in projector plugin, offset_x and offset _y but seems implement in plugin camera, So I wonder how can I change  the offset_x and offset_y in projector plugin

**Comment by njroussel** [comment]
Hi @ljijjiang1234 

I might be mistaken, or misunderstanding what you are trying to achieve, but translating the projector within its plane should produce the same results.
If you actually want to crop parts of the projected texture by offsetting it, you're better off directly modifying your `Texture` object.
